sha2/src/sha256_aarch64.S

```diff
@@ -0,0 +1,277 @@
+/*
+ * SHA-256 hash in AArch64 assembly
+ *
+ * Copyright (c) 2020 Emmanuel Gil Peyrot <linkmauve@linkmauve.fr>. (MIT License)
```

Comment:
Can you document the provenance of this assembly (and also SHA-1), i.e. a link to the repo where it was fetched from?

Reply:
I wrote it myself, hence this copyright.

I had a look at this repository to understand how to use the instructions, but that’s about it: https://github.com/noloader/SHA-Intrinsics

Reply:
Aha, well then!